drivers/gpio/gpio_mcux_igpio.c

```diff
@@ -121,6 +121,18 @@ static int mcux_igpio_pin_interrupt_configure(struct device *dev,
 	u8_t icr;
 	int shift;
 
+	if (mode == GPIO_INT_MODE_DISABLED) {
+		key = irq_lock();
+
+		WRITE_BIT(base->ISR, pin, 1);
+		WRITE_BIT(base->IMR, pin, 0);
+		WRITE_BIT(data->pin_callback_enables, pin, 0);
+
+		irq_unlock(key);
+
+		return 0;
+	}
+
```

Comment:
Since the `mode == GPIO_INT_MODE_DISABLED` case is handled here and then we exit the function all the occurences of `mode != GPIO_INT_MODE_DISABLED` at the bottom of this function (Github doesn't allow me to place this comment where it should be located) should be replaced with 1. i.e. s/mode != GPIO_INT_MODE_DISABLED/1/

Reply:
Thanks for your review, Do you means replace
```
WRITE_BIT(base->ISR, pin, mode != GPIO_INT_MODE_DISABLED);
WRITE_BIT(base->IMR, pin, mode != GPIO_INT_MODE_DISABLED);
WRITE_BIT(data->pin_callback_enables, pin,  mode != GPIO_INT_MODE_DISABLED);
```
to
```
WRITE_BIT(base->ISR, pin, 1);
WRITE_BIT(base->IMR, pin, 1);
WRITE_BIT(data->pin_callback_enables, pin,  1);
```